docs/faq.md

```diff
@@ -33,7 +33,16 @@ library(SingleCellExperiment)
 scpca_sample <- readRDS("SCPCL000000_filtered.rds")
 ```
 
-## What is the difference between a sample and a library?
+## What is the difference between participants, samples, and libraries?
+
+The `participant_id` indicates the participant, or patient, from which a sample, or collection of samples, is obtained from. 
+A sample ID, labeled as `scpca_sample_id` and indicated by the prefix `SCPCS`, represents a unique tissue that was collected from a participant. 
+For example, one participant may have a sample collected both at initial diagnosis and at relapse.
+This would result in two unique sample ID's with the same participant ID. 
+
+The library ID, labeled as `scpca_library_id` and indicated by the prefix `SCPCL`, represents a single emulsion and droplet generation using the 10X Genomics workflow from a given sample. 
```

Comment:
I think we want to make this a bit less specific, as we use library ids also for bulk, so we should to say something about that. I'm fighting with it a bit here, but trying to get across that the CITE-seq sequencing library falls in the same library, but bulk does not. I don't know it "set of cells" is a good metric, but it is what I came up with...


```suggestion
The library ID, labeled as `scpca_library_id` and indicated by the prefix `SCPCL`, represents a single set of cells from a tissue sample.
For a bulk RNA-seq experiment, this will result in a single sequencing library. 
For single-cell or single-nuclei experiments, this will be the result of emulsion and droplet generation using the 10X Genomics workflow, potentially including both RNA-seq, CITE-seq and cell hashing sequencing libraries. 
```